_config/synonyms.yml

```diff
@@ -0,0 +1,34 @@
+---
+Name: forager-bifrost-synonyms
+Only:
+  envvarset: 'BIFROST_MANAGEMENT_API_KEY'
+After:
+  - 'silverstripe-forager-elastic-enterprise-synonyms'
+  - 'silverstripe-forager-synonyms'
+---
+SilverStripe\Core\Injector\Injector:
+  # Adaptors provided by this module
+  SilverStripe\Forager\Interfaces\Requests\CreateSynonymRuleAdaptor:
+    class: SilverStripe\ForagerBifrost\Adaptors\Requests\CreateSynonymRuleAdaptor
+  SilverStripe\Forager\Interfaces\Requests\GetSynonymRuleAdaptor:
+    class: SilverStripe\ForagerBifrost\Adaptors\Requests\GetSynonymRuleAdaptor
+  SilverStripe\Forager\Interfaces\Requests\GetSynonymRulesAdaptor:
+    class: SilverStripe\ForagerBifrost\Adaptors\Requests\GetSynonymRulesAdaptor
+  SilverStripe\Forager\Interfaces\Requests\UpdateSynonymRuleAdaptor:
+    class: SilverStripe\ForagerBifrost\Adaptors\Requests\UpdateSynonymRuleAdaptor
+  SilverStripe\Forager\Interfaces\Requests\DeleteSynonymRuleAdaptor:
+    class: SilverStripe\ForagerBifrost\Adaptors\Requests\DeleteSynonymRuleAdaptor
+  # Adaptors provided by the ElasticEnterprise dependency
+  SilverStripe\Forager\Interfaces\Requests\GetSynonymCollectionsAdaptor:
```

Comment:
could we re-implement this to decouple the modules a bit more? Seems a simple one

Reply:
These modules will be completely decoupled for the CMS 6 version - at the moment, this is just following the existing paradigm of using Elastic Enterprise wherever possible (so that we effectively cover support for both).